src/components/Button/__stories__/button.stories.mdx

```diff
@@ -228,4 +228,4 @@ When one action is selected, the other is disabled
 </Canvas>
 
 ## Related components
-<RelatedComponents componentsNames={[SPLIT_BUTTON, BUTTON_GROUP, BUTTON_GROUP]}/>
\ No newline at end of file
+<RelatedComponents componentsNames={[SPLIT_BUTTON, BUTTON_GROUP, CHIPS]}/>
```

Comment:
let's change the name of the "chips" constant to CHIP: "chip" because in the near future we will change the component name (please do not change the component name for now, only the constant and the name of the component description component)